src/core/src/bootstrap/Bootstrapper.py

```diff
@@ -142,36 +143,46 @@ def basic_environment_health_check(self):
         self.composite_logger.log("Process id: " + str(os.getpid()))
 
         # Ensure sudo works in the environment
-        sudo_check_result = self.check_sudo_status()
+        sudo_check_result = self.check_sudo_status_retry()
         self.composite_logger.log_debug("Sudo status check: " + str(sudo_check_result) + "\n")
 
+    def check_sudo_status_retry(self, raise_if_not_sudo=True):
```

Comment:
See name of the function in the previous comment:
![image](https://github.com/user-attachments/assets/11e2877f-c4c4-42e7-98be-d4ebca0449ed)

Reply:
revert check_sudo_status to original state, but i think we should move this self.composite_logger.log("Performing sudo status check... This should complete within 10 seconds.") to retry logic